apps/laboratory/tests/wallet.spec.ts

```diff
@@ -45,6 +45,12 @@ sampleWalletTest('it should fetch balance as expected', async ({ library }) => {
   await modalValidator.expectBalanceFetched(library === 'solana' ? 'SOL' : 'ETH')
 })
 
+sampleWalletTest('it should show onramp button accordingly', async () => {
+  await modalPage.openModal()
+  await modalValidator.expectOnrampButton()
+  await modalPage.closeModal()
+})
+
```

Comment:
@enesozturk unsure why but this was after the unsupported network test , causing it to fail

Reply:
Maybe it's bc we see unsupported network view and it cannot open the account view? Need to repro locally to see what's off, let me take a look after review please